extensions/reactive-pg-client/runtime/src/main/java/io/quarkus/reactive/pg/client/runtime/DataSourceReactivePostgreSQLConfig.java

```diff
@@ -23,4 +23,13 @@ public interface DataSourceReactivePostgreSQLConfig {
      */
     @ConfigDocDefault("disable")
     Optional<SslMode> sslMode();
+
+    /**
+     * Level 7 proxies can load balance queries on several connections to the actual database.
+     * When it happens, the client can be confused by the lack of session affinity and unwanted errors can happen like
+     * ERROR: unnamed prepared statement does not exist (26000).
+     * See <a href="https://vertx.io/docs/vertx-pg-client/java/#_using_a_level_7_proxy" </a>
+     */
+    @ConfigDocDefault("false")
+    Optional<Boolean> useLayer7Proxy();
```

Comment:
You have a default value, so let's not make it `Optional` given you always have a value.
```suggestion
    boolean useLayer7Proxy();
```

Reply:
Sure, I just made the change!